source/agora/consensus/validation/Block.d

```diff
@@ -111,9 +111,15 @@ public string isInvalidReason (in Block block, Engine engine, Height prev_height
     if (block.header.height == 0)
         return "Block: Genesis block should be validated using isGenesisBlockInvalidReason";
 
-    assert(random_seed !is Hash.init);
-    // Validate this after the genesis check for better UX
-    assert(prev_hash !is Hash.init);
+    version (unittest)
+    {
+        // FIXME: Remove this skip of random seed checks in unit tests
+    } else
```

Comment:
As long as this is removed soon, I guess it's fine.

Reply:
The plan is to remove this when the pre-images are made available to the test code which will need to be the case for PR https://github.com/bosagora/agora/pull/1860